scenes/eternal_loom_sokoban/levels/sokoban_level_2b.tscn

```diff
@@ -0,0 +1,57 @@
+[gd_scene load_steps=5 format=4 uid="uid://c6qwiy01e6jqd"]
+
+[ext_resource type="Script" uid="uid://dppvw2f8yh4vu" path="res://scenes/eternal_loom_sokoban/components/system/board/board_2d.gd" id="1_k8gxf"]
+[ext_resource type="Material" uid="uid://jpl3v3f7elq6" path="res://scenes/eternal_loom_sokoban/components/floor/tiled_mat.tres" id="2_2kj74"]
+[ext_resource type="TileSet" uid="uid://dand23uvn70pg" path="res://scenes/eternal_loom_sokoban/components/eternal_loom_tileset.tres" id="2_p3sau"]
+[ext_resource type="PackedScene" uid="uid://btpeaqx2nur3q" path="res://scenes/eternal_loom_sokoban/components/eternal_loom_ruleset.tscn" id="4_vc6vr"]
+
+[node name="SokobanLevel2b" type="Node2D"]
```

Comment:
I'm not very good at this game but, is this really solvable?!

![image](https://github.com/user-attachments/assets/0d08ead5-265a-47ec-9f17-23a2812df72d)

The half-spools are barriers so I can't see how it's possible to get through to the left-hand side without permanently blocking the yellow thread goal.

Reply:
Good catch! I broke this puzzle while adding the threads! I'll fix that one.